rust_src/remacs-sys/lib.rs

```diff
@@ -1043,7 +1043,7 @@ extern "C" {
 }
 
 extern "C" {
-    pub fn pget_raw_status_new(p: *const Lisp_Process) -> c_int;
+    pub fn pget_raw_status_new(p: *const Lisp_Process) -> BoolBF;
```

Comment:
Just use bool.

Reply:
Why did we introduce BoolBF if we can use bool ?

Reply:
I believe because it was needed to access the bit fields. But there is no reason to expose that the underlying implementation is a bit field here.

Reply:
It seems there were reasons for using BoolBF #430.

Reply:
Yes, there were reasons. But in this code there is no reason to export the value as a bit field. All we need is a boolean YES/NO.

Reply:
`raw_status_new` is a bool_bf like the other struct members where we use BoolBF. What am I missing here ?

Reply:
Write a `bool_bf` but expose a `bool` to Rust. As far as Rust is concerned there is no difference between the two.

Reply:
So this is like every other case where we use BoolBF. I would suggest that we stay consistent with it and change all occurrences then. It's already confusing enough for newcomers.